proyecto-frontend/src/app/modules/product/component/category/category.component.ts

```diff
@@ -24,6 +24,7 @@ export class CategoryComponent implements OnInit {
   swal = Swal;
   isActiveList: boolean = false;
   showAllCategories: boolean = false;
```

Comment:
Comienzas mostrando todas las categorías
```suggestion
  showAllCategories: boolean = true;
```